fenix/app/src/androidTest/java/org/mozilla/fenix/ui/ext/ComposeTestRule.kt

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ui.ext
+
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onNodeWithTag
+import org.mozilla.fenix.tabstray.TabsTrayTestTag
+
+/**
+ * Obtains the root Tabs Tray.
+ */
+fun ComposeTestRule.tabsTray() = onNodeWithTag(TabsTrayTestTag.tabsTray)
```

Comment:
I don't think we need this extra step. You can just use composeTestRule.onNodeWithTag(TabsTrayTestTag.tabsTray) directly in the robot class.